SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/HomeViewController.swift

```diff
@@ -33,30 +48,54 @@ class HomeViewController: UIViewController {
         collectionView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
         collectionView.register(UINib(nibName: "HomeCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "mycell")
         
+        testimoniosCollectionView.isPagingEnabled = true
+        testimoniosCollectionView.dataSource = self
+        testimoniosCollectionView.delegate = self
+        testimoniosCollectionView.contentInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
+        testimoniosCollectionView.register(UINib(nibName: "TestimoniosCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "Tcell")
+        
     }
     
 }
+
 extension HomeViewController: UICollectionViewDataSource {
     
     func numberOfSections(in collectionView: UICollectionView) -> Int {
         return 1
     }
-    
+
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return sliderData.count
+        switch collectionView {
+        case self.testimoniosCollectionView:
+            // Return Max pages = 4
+            return testimoniosData.count > 4 ? 4 : testimoniosData.count
```

Comment:
read the history in jira " Este slider deberá tener una capacidad máxima de 4 y como último elemento una flecha que dará acción a abrir una nueva pantalla (no es objetivo de esta historia)."  thats 4 elements plus the arrow.  You ask and I literally said that is anotar item,  and you should not ever use a button in a cell when you can react to the selection

Reply:
But that i did with the four pages is okey right?